src/components/sections/secondary-contact-section.astro

```diff
@@ -0,0 +1,30 @@
+---
+import TilePatternBgBlack from "../backgrounds/tile-pattern-bg-black.astro";
+import BookingWhiteButton from "../buttons/booking-white-button.astro";
+import SectionHeader from "../common/section-header.astro";
+import Section from "../common/section.astro";
+import WhiteSpan from "../common/white-span.astro";
+---
+
+<div class="relative z-0 bg-background-dark">
+  <TilePatternBgBlack />
+
+  <div class="-mb-40 py-28">
+    <Section>
+      <div class="relative flex flex-col gap-10 py-20">
+        <SectionHeader>
+          <span slot="title" class="flex flex-col">
+            <WhiteSpan>Don’t think twice </WhiteSpan>
+            <WhiteSpan>let’s bring your vision to life!</WhiteSpan>
+          </span>
```

Comment:
I like the idea of creating and using atomic components, but I find this `WhiteSpan` unnecessary 🤔. We could use normal `span`s, make the parent span (the one with `slot='title'`) have the class `text-white dark:text-white` and the child spans will inherit it.

I think we should atomize when possible, but also only if it worth it and makes sense doing the effort.

```html
<span slot="title" class="flex flex-col text-white dark:text-white">
    <span>Don’t think twice</span>
    <span>Let’s bring your vision to life!</span>
</span>
```